src/main/java/lotto/domain/LottoTicketSeller.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoTicketSeller {
+
+    public List<LottoTicket> generateTickets(LottoPurchaseBudget budget) {
+        int ticketQuantity = budget.getTicketQuantity();
+
+        List<LottoTicket> tickets = new ArrayList<>();
+        List<LottoNumber> lottoNumbers = LottoNumber.getValues();
+        for (int i = 0; i < ticketQuantity; i++) {
+            Collections.shuffle(lottoNumbers);
+            tickets.add(new LottoTicket(lottoNumbers.subList(0, 6)));
+        }
+
+        return tickets;
```

Comment:
`0`, `6` 같은 숫자는 [매직 넘버](https://javabom.tistory.com/28)로 보이네요. 
상수로 추출해서 의미를 드러내주면 좋을 것 같아요. 🙂

```suggestion
        return Stream.generate(() -> new LottoTicket(lottoNumbers.subList(0, 6)))
            .limit(ticketQuantity)
            .collect(Collectors.toList());
```